synchrad/calc.py

```diff
@@ -1,7 +1,8 @@
 import numpy as np
 import h5py
 import pyopencl as cl
-cl.create_some_context()
```

Comment:
I commented this out since it didn't really seem necessary. Can we remove it completely?

Reply:
this line was needed to handle some strange bug that I had on my mac laptop (AMD card) with their weird driver, which had to be pre-initialized at this point. I do not used this machine now, so we can keep it commented for now and see if there will be any issues with that.